lib/date.nit

```diff
@@ -0,0 +1,254 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date and Time manipulation
+# The class Date hase the basic opperation to use only the date data
+# The class Time hase the basic opperation to use only the time data
+# The class DateTime extend the class Date and Time to use the two of date and time data
+#
+# 	var p_date = new Date.set(2014, 01, 09)
+#	var auto_date = new Date.today ==> set the current date 
+#	var auto_time = new Time.now ==> set the current time
+#	var date_time = new DateTime.today ==> set the current date and time
+#
+#	#################
+#	#Some opperation#
+#	#################
+#
+#	#DateTime:
+#	var date_1  = new DateTime.time(2014, 02, 09, 13, 50, 0)
+#	var date_2  = new DateTime.time(2014, 02, 09, 13, 51, 0)
+#	
+#	date_1 > date_2 ==> return false
+# 	
+#	#Date:
+#	var date_1 = new Date.set(2014, 01, 12)
+#	var date_2 = new Date.set(2014, 01, 30)
+#	
+#	date_1.diff_month(date_2) ==> retrun 0
+#
+#	#Time:
+#	var time_1 = new Time.now
+#	var time_2 = new Time.set(14,25,30)
+#	
+#	time_1 == time_2
+#
+module date
+
+# The class "Time" allow the manipulation only on the time data
```

Comment:
Rather than stating the limitations, you might prefer a more positive tone.
Something along the lines of "`Time` handles all time-related data (hours, minutes, seconds)"